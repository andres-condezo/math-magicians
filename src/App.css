.content {
  background-color: grey;
  width: 100%;
  height: 100%;
  display: flex;
  flex-direction: column;
  justify-content: flex-start;
  align-items: center;
  padding-top: 2rem;
}
