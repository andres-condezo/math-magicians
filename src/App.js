import React, { Component } from 'react';
import {
  BrowserRouter as Router,
  Routes,
  Route,
} from 'react-router-dom';

import './App.css';
import Home from './components/Home';
import Calculator from './components/Calculator';
import Quote from './components/Quote';
import NavBar from './components/NavBar';

class App extends Component {
  constructor(props) {
    super(props);
    this.state = {};
  }

  render() {
    return (
      <>
        <Router>
          <NavBar />
          <section className="content">
            <Routes>
              <Route
                path="/math-magicians"
                element={<Home />}
              />
              <Route
                path="/Calculator"
                element={<Calculator className="calculator" />}
              />
              <Route
                path="/Quote"
                element={<Quote />}
              />
            </Routes>
          </section>
        </Router>
      </>
    );
  }
}

export default App;
