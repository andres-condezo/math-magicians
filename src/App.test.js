import React from 'react';
import renderer from 'react-test-renderer';
import App from './App';

// test('renders learn react link', () => {
//   render(<App />);
//   const linkElement = screen.getByText(/learn react/i);
//   expect(linkElement).toBeInTheDocument();

test('match snapshot', () => {
  const tree = renderer.create(<App />).toJSON();
  expect(tree).toMatchSnapshot();
});
