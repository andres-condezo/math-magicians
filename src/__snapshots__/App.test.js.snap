// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`match snapshot 1`] = `
Array [
  <nav>
    <div>
      <a
        className="logo"
        href="/math-magicians"
        onClick={[Function]}
      >
        Math Magicians
      </a>
    </div>
    <ul>
      <li>
        <a
          className=""
          href="/math-magicians"
          onClick={[Function]}
        >
          Home
        </a>
      </li>
      <li>
        <a
          className=""
          href="/Calculator"
          onClick={[Function]}
        >
          Calculator
        </a>
      </li>
      <li>
        <a
          className=""
          href="/Quote"
          onClick={[Function]}
        >
          Quote
        </a>
      </li>
    </ul>
  </nav>,
  <section
    className="content"
  />,
]
`;
